### PR TITLE
Updated model for the anomaly_stfpm_quantize_with_accuracy_control example

### PR DESCRIPTION
**Pull request by alexsu52**
### Changes

Updated the model used in the anomaly_stfpm_quantize_with_accuracy_control example to a more accurate model.

### Reason for changes

Demonstrate quantize with accuracy control flow

### Related tickets

ref: 106970

### Tests

test_examples.py

### CODE REVIEW

File: examples/post_training_quantization/openvino/anomaly_stfpm_quantize_with_accuracy_control/main.py [hunk 1]
```diff
@@ -64,6 +64,7 @@ def get_anomaly_images(data_loader: Iterable[Any]) -> List[Dict[str, torch.Tenso
 def validate(model: ov.CompiledModel, val_loader: Iterable[Any], val_params: Dict[str, float]) -> float:
```
Comment (andrey-churkin) [inline examples/post_training_quantization/openvino/anomaly_stfpm_quantize_with_accuracy_control/main.py:64]: Please update return type hint

Comment (alexsu52) [reply examples/post_training_quantization/openvino/anomaly_stfpm_quantize_with_accuracy_control/main.py]: Done

**Comment by andrey-churkin** [comment]
run pytorch pre-commit tests

**Review by andrey-churkin — APPROVED** [review]

**Comment by alexsu52** [comment]
run pytorch pre-commit tests

**Pull request merged**